multi-node-testkit/src/main/mima-filters/1.0.1.backwards.excludes/migrate-to-netty4.backwards.excludes

```diff
@@ -0,0 +1,57 @@
+# Migrate to netty 4
+ProblemFilters.exclude[IncompatibleTemplateDefProblem]("org.apache.pekko.remote.testconductor.RemoteConnection")
```

Comment:
I would consider this as not something that is needed for 1.0.1 and that should be targeted at 1.1.0

Reply:
you can't make major changes to lib dependencies in a patch release

Reply:
This is not a part of user facing api, maybe this is ok?

Reply:
* pekko-multi-node-testkit is a public jar
* it is my opinion that no public jar should have a major version increase in a patch release
* I'm quite happy for this change to be added but just not to 1.0.1

Reply:
I will update the mima filter then.